### ISSUE TITLE
Channel reserve message? 

### ISSUE THREAD

**Issue by kingonly** [breez/c-breez#597]
![Screenshot_2023-07-11-16-38-31-00_9a3a9060ca72a38f8a1fec1d0c1a640e](https://github.com/breez/c-breez/assets/31890660/43ceabfe-1052-4eaa-913c-cffe8c5e618a)

I had is for a while, but now it's gone

**Comment by erdemyerebasmaz** [comment]
Yes @kingonly, that is the channel reserve message. It exists on breezmobile as well.

**Comment by kingonly** [comment]
@erdemyerebasmaz 
1. We set the channel reserve to zero.
2. As I wrote, the message disappeared after while. There's something strange going on.

**Comment by JssDWt** [comment]
I think this has to do with funds that are reserved for fees or htlcs. That will reduce `maxPayableMsat`.We calculate the 'reserve' by substracting the channel balance with `maxPayableMsat`. But that may not always be correct I think, or it might update too slow. The account state sometimes doesn't update very quickly after a payment it seems.

**Comment by JssDWt** [comment]
@roeierez I think it's mostly the account state that updates too slowly.
July 9th @Nackoo2000 reported a big delay in the balance screen update.
June 29th @Nackoo2000 reported a similar message as the one in this issue. That one was due to a htlc being in-flight. Or a htlc that had been in flight shortly before. He had a 'channel reserve' with the size of approximately the payment amount. So I think that was the htlc itself that was counted as reserve.

**Comment by roeierez** [comment]
Makes sense. So the htlc is still in-flight in greenlight despite payment succeeded?
We do sync immediately after payment success but the fact that @Nackoo2000 reported "payment not found" makes me think it is pending because although we sync all payments (including pending) our lookup looks for completed payment only (https://github.com/breez/breez-sdk/blob/main/libs/sdk-core/src/breez_services.rs#L603).
Probably that needs to be changed at first step (include pending payments in the lookup), what do you think?
cc @ok300

**Comment by kingonly** [comment]
The payment was definitely successful in my case.
Regardless, I think we should just remove this message from the UI.
